app/utils/Utils.ts

```diff
@@ -246,7 +242,7 @@ export default class Utils {
     // we need to exclude 2 use cases:
     // 1. send to self (make no sense to do a donation here)
```

Comment:
i think this comment is outdated now?

Reply:
upsss, yes. I'm going to fix in the next PR asap.